src/php/views/base.twig

```diff
@@ -1,26 +1,27 @@
 {% include 'header.twig' %}
 
-		<article id="post-{{ post.id }}" role="main" class="post-type-{{ post.post_type }} {{ function('get_post_class')|join(' ') }}">
+<article id="post-{{ post.id }}" class="post-type-{{ post.post_type }} {{ function('get_post_class')|join(' ') }}">
 
-			{% if post.title %}
-        <div class="obj-width-limiter">
-          {# equivalent to the_title() #}
-          <h1>{{ post.title }}</h1>
-        </div>
-      {% endif %}
+	{% if post.title %}
+		<div
+			class="obj-width-limiter">
```

Comment:
Is there a reason the `class` attribute is on a different line here?

Reply:
That was automatic formatting from a Twig extension I had installed in my editor. I agree that it doesn't make much sense, so I disabled it and updated that line.